samples/multicluster/setup-mesh.sh

```diff
@@ -0,0 +1,432 @@
+#!/bin/bash
+
+# Copyright 2018 Istio Authors
+#
```

Comment:
I'm uncomfortable with mixing bookinfo into this. I'd like to have a clean "setup for multicluster" story, independent of the bookinfo demo. Could we just have two scripts instead?

Reply:
done